### PR TITLE
Change next release version from 7.0.1 to 7.1.0 to avoid confusion

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Change future release version to `7.1.0` so that no one thinks that these changes require release for Kiwi Bugfix

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Change history for stripes-smart-components
 
-## [7.0.1] (IN PROGRESS)
+## [7.1.0] (IN PROGRESS)
```
Comment (AndreiShumski2) [inline CHANGELOG.md:3]: IMO it's better to remove the version at all - `## (IN PROGRESS)`. And put release version only during new release creation.

**Review by zburke — COMMENTED** [review]
I _do_ need to do for _all_ the stripes-* repos, but it needs to happen in order from the bottom of the hierarchy (stripes-components) up to the top (stripes). Sorry for the delay on that. I'll try to get that done, including this PR, Monday or Tuesday.

**Comment by BogdanDenis** [comment]
> I _do_ need to do for _all_ the stripes-* repos, but it needs to happen in order from the bottom of the hierarchy (stripes-components) up to the top (stripes). Sorry for the delay on that. I'll try to get that done, including this PR, Monday or Tuesday.

Thanks, Zak
While we're on topic of module versions - I think we should also increment `eslint-config-stripes` to `6.1.0`. I saw that you reverted changes with webpack upgrade and with that revert the version went from `6.1.0` to `6.0.0`. But the `latest` still points to `6.1.0` which causes errors in GitHub actions tests

**Comment by zburke** [comment]
Oooof, thanks for bringing the bad eslint-config-stripes releases to my attention. I think I got that sorted out (at least for npm-folioci; I still need to do the same for npm-folio), but some tests are still failing here and in stripes-core. The fun never stops! I'll merge this as soon as I figure out why these tests started dying.

**Comment by zburke** [comment]
@BogdanDenis , at this point I think this PR is blocked on broken tests (which @NikitaSedyx has fixed as part of #1140), not on `eslint-config-stripes` problems.

File: CHANGELOG.md (hunk 1 above)
Comment (NikitaSedyx) [reply CHANGELOG.md:3]: in acq we keep it's always in progress at the top and add release version when do releases

https://github.com/folio-org/ui-orders/blob/master/CHANGELOG.md#in-progress

**Review by NikitaSedyx — APPROVED** [review]

**Comment by NikitaSedyx** [comment]
@BogdanDenis test fixes are merged so your request can be unblocked

**Pull request merged**